frontend/src/components/TaskRelationTable.tsx

```diff
@@ -174,13 +171,13 @@ const relationTable: (def: RelationTableDef) => FC<RelationTableProps> = ({
           itemSize={(idx) => rowHeights[idx] ?? 0}
           itemCount={tasks.length}
           height={Math.min(height, listHeight)}
-          width={editMode ? '105%' : '100%'}
+          width="100%"
```

Comment:
```suggestion
          width="105%"
```

The extra room here is still needed, so that the remove button doesn't cause horizontal scroll